website/docs/core/certificates.md

```diff
@@ -67,15 +67,15 @@ ak import_certificate --certificate /certs/mycert.pem --private-key /certs/somet
 # ak import_certificate --certificate /certs/othercert.pem --name test2
 ```
 
-This will import the certificate into authentik under the given name. This command is idempotent, meaning you can run it via a cron-job and authentik will only update the certificate when it changes.
+This will import the certificate into authentik under the given name. This command is idempotent, meaning you can run it via a cron job and authentik will only update the certificate when it changes.
```

Comment:
Suggestion: "This command is safe to run as a cron job. Authentik will only re-import the certificate if it changes." The "idempotent" is good for programmers, but probably not that familiar to admins.